### DISCUSSION TITLE
Regarding the issue of plot

### DISCUSSION THREAD

**Discussion by Santiagopeacely** [pybop-team/PyBOP#604] [Q&A] [answered]
Dear Pybop:
  Hello, may I ask if the visualization module will continue to use Go or will use matplotlib as the visualization tool in the next version?

**Comment by BradyPlanden** [comment, marked as answer]
Hi @Santiagopeacely,

The standard plot methods offered by PyBOP will continue using Plotly for the next release. As PyBOP ships with PyBaMM, matplotlib will also be installed, so you can decide between the two packages for any additional plots you wish to create.

**Comment by Santiagopeacely** [comment]
Thank you very much！
